resolwe/flow/utils/test.py

```diff
@@ -224,6 +224,8 @@ def tearDown(self):
                 shutil.rmtree(data_dir, ignore_errors=True)
                 shutil.rmtree(export_dir, ignore_errors=True)
 
+        Process.objects.all().delete()
```

Comment:
For each test we created 2 new users in the database:
- https://github.com/genialis/resolwe/blob/master/resolwe/flow/utils/test.py#L197
- https://github.com/genialis/resolwe/blob/master/resolwe/flow/tests/test_executors.py#L57

I guess the auto ID field was not reset after each test.